README.md

```diff
@@ -491,13 +497,56 @@ manager.getCredentials(new BaseCallback<Credentials, CredentialsManagerException
 ```
 
 
-5. **Clear credentials**
+5. **Clear credentials:**
 When you want to log the user out:
 
 ```java
 manager.clearCredentials();
 ```
 
+
+### Encryption enforced (Min API 21)
+
+The enhanced version contains the same methods as the _Basic_ manager but encrypts the data before storing it, and in those devices where a Secure LockScreen has been configured it can require the user authentication before letting them obtain the stored credentials. The class is called `CryptoManager`. 
```

Comment:
The enhanced version contains the same methods as the _Basic_ manager but encrypts the data before storing it, and in ->
This version expands the minimum version and adds encryption to the data storage. In